packages/cadl-python/test/generated/hello/hello/_operations/_operations.py

```diff
@@ -84,12 +86,9 @@ def world(self, **kwargs: Any) -> str:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
-        if response.content:
-            deserialized = response.json()
-        else:
-            deserialized = None
+        deserialized = _deserialize(None, response.json())
```

Comment:
@iscai-msft @changlong-liu Some operation folder's name is `operations` while some is `_operations`. What is the reason of  the difference?
![image](https://user-images.githubusercontent.com/70930885/190582599-6656b4aa-e18d-4cc2-bda5-9395f61eecb0.png)

Reply:
should be related to logic here: https://github.com/Azure/autorest.python/blob/20220705-dpg-model/packages/autorest.python/autorest/codegen/models/code_model.py#L194 

I guess by this way we are encouraging people to use operation directly from `client` when it's `mixin`?

Reply:
Ideally we don't want to expose operation group namespaces to users. Users should only be accessing them through the client. When we expose the operations namespace, people sometimes get confused and initialize the operation groups themselves. However, for documentation purposes, we need to expose them if they are a separate operation group, otherwise we won't be able to show all of the operations on each operation group. For mixin operations though, those are directly on the client, so we don't need to expose the operations namespace, so in that case, we generate the folder as private

Reply:
Thanks for the detailed explanation !